File: recaudacion/2.Codigo/frontend/src/app/core/models/ingresopecosa.ts
export class IngresoPecosa {
    ingresoPecosaId: number;
    unidadEjecutoraId: number;
    tipoDocumentoId: number;
    anioPecosa: number;
    tipoBien: string;
    numeroPecosa: number;
    fechaPecosa: any;
    nombreAlmacen: string;
    motivoPedido: string;
    fechaRegistro: any;
    estado: number;
    estadoNombre: string;
    usuarioCreador: string;
    usuarioModificador: string;
    ingresoPecosaDetalle: IngresoPecosaDetalle[];
    index:number;
}

export class IngresoPecosaEstado {
    ingresoPecosaId: number;
    estado: number;
    usuarioModificador: string;
}

export class IngresoPecosaFilter {
    pageNumber: number;
    pageSize: number;
    sortColumn:string;
    sortOrder: string;
    unidadEjecutoraId: number;
    anioPecosa: number;
    tipoBien: string;
    numeroPecosa: number;
    fechaInicio: any;
    fechaFin: any;
    estado: number;
    rol: string;
}


export class IngresoPecosaDetalle {
    ingresoPecosaDetalleId: number;
    ingresoPecosaId: number;
    anioPecosa: number;//ops
    numeroPecosa: number; //ops
    catalogoBienId: number;
    unidadMedida: string;
    codigoItem: string;
    nombreItem: string;
    nombreMarca: string;
    cantidad: number;
    cantidadSalida: number;
    saldo: number;//ops
    precioUnitario: number;
    valorTotal: number;
    serieFormato: string;
    serieDel: number;
    serieAl: number;
    serieDelSalida: number;//ops
    serieAlSalida: number;//ops
    estado: string;
    usuarioCreador: string;
    usuarioModificador: string;
    index:number;
}

File: recaudacion/2.Codigo/backend/RecaudacionApiIngresoPecosa/Domain/IngresoPecosa.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace RecaudacionApiIngresoPecosa.Domain
{
    [Table("INGRESO_PECOSA")]
    public class IngresoPecosa
    {
        [Key]
        [Column("INGRESO_PECOSA_ID")]
        public int IngresoPecosaId { get; set; }
        [Column("UNIDAD_EJECUTORA_ID")]
        public int UnidadEjecutoraId { get; set; }
        [Column("TIPO_DOCUMENTO_ID")]
        public int TipoDocumentoId { get; set; }
        [Column("INGRESO_PECOSA_ANIO")]
        public int AnioPecosa { get; set; }
        [Column("INGRESO_PECOSA_TIPO_BIEN")]
        public string TipoBien { get; set; }
        [Column("INGRESO_PECOSA_NUMERO")]
        public int NumeroPecosa { get; set; }
        [Column("INGRESO_PECOSA_FECHA")]
        public DateTime FechaPecosa { get; set; }
        [Column("INGRESO_PECOSA_NOMBRE_ALMACEN")]
        public string NombreAlmacen { get; set; }
        [Column("INGRESO_PECOSA_MOTIVO_PEDIDO")]
        public string MotivoPedido { get; set; }
        [Column("INGRESO_PECOSA_FECHA_REGISTRO")]
        public DateTime FechaRegistro { get; set; }
        [Column("INGRESO_PECOSA_ESTADO")]
        public int Estado { get; set; }
        [NotMapped]
        public string EstadoNombre { get; set; }
        [Column("USUARIO_CREADOR")]
        public string UsuarioCreador { get; set; }
        [Column("FECHA_CREACION")]
        public DateTime? FechaCreacion { get; set; }
        [Column("USUARIO_MODIFICADOR")]
        public string UsuarioModificador { get; set; }
        [Column("FECHA_MODIFICACION")]
        public DateTime? FechaModificacion { get; set; }
    }

}
